data/core/units/dunefolk/Alchemist.cfg

```diff
@@ -0,0 +1,80 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Dune Alchemist
+    name= _ "Dune Alchemist"
+    race=dunefolk
+    image="units/dunefolk/alchemist.png"
+    profile="portraits/dunefolk/herbalist.png"
+    hitpoints=41
+    movement_type=dunefoot
+    movement=5
+    experience=100
+    level=2
+    alignment=liminal
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=33
+    usage=fighter
+    [abilities]
+        {ABILITY_REGENERATES}
+    [/abilities]
+    description= _ " Concept by Krogen
```

Comment:
Should mainline have author in description?

Reply:
No.

Reply:
The descriptions are still to be reviewed by Yumi/Elynia/Namaera and that is why I believe some indicate that they have been written but not yet reviewed. However, I believe "TODO" would have been a more appropriate message.